sql/moz-fx-data-shared-prod/mozilla_vpn/events_unnested/view.sql

```diff
@@ -0,0 +1,29 @@
+-- Created by hand (rather than via ./bqetl glean_usage generate)
+-- because the VPN doesn't have an events ping (just a "main" one,
+-- in which there are events)
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.mozilla_vpn.events_unnested`
+AS
+SELECT
+  e.* EXCEPT (events, metrics) REPLACE(
+    "release" AS normalized_channel,
+        -- Order of ping_info fields differs between tables; we're verbose here for compatibility
+    STRUCT(
+      ping_info.end_time,
+      ping_info.experiments,
+      ping_info.ping_type,
+      ping_info.seq,
+      ping_info.start_time,
+      ping_info.reason,
+      ping_info.parsed_start_time,
+      ping_info.parsed_end_time
+    ) AS ping_info
+  ),
+  event.timestamp AS event_timestamp,
+  event.category AS event_category,
+  event.name AS event_name,
+  event.extra AS event_extra,
+FROM
+  `moz-fx-data-shared-prod.mozillavpn.main` AS e
```

Comment:
```suggestion
  `moz-fx-data-shared-prod.mozilla_vpn.main` AS e
```

I guess `mozilla_vpn.main` does reference `mozillavpn.main`, but it might be less confusing to use `mozilla_vpn` here. I thought at first `mozillavpn` is a typo. I guess for some historical reasons there are two vpn datasets.

Reply:
Agreed. `mozillavpn` refers to the app id but using the application-wide dataset makes more sense here, I think. 

https://github.com/mozilla/probe-scraper/blob/e6725dca9ed9b97d8cb27762a69b98498e8d8552/repositories.yaml#L691